sunpy_sphinx_theme/conf.py

```diff
@@ -36,23 +36,14 @@ def page_url(page):
             "About",
             [
                 ("Our Mission", page_url("about.html"), 1),
-                (
-                    "Acknowledge SunPy",
-                    page_url("about.html") + "#acknowledging-or-citing-sunpy",
-                    1,
-                ),
-                (
-                    "Code of Conduct",
-                    page_url("coc.html"),
-                    1,
-                ),
+                ("Project Structure", page_url("project/"), 1)
```

Comment:
I don't think "project structure" is a good title. This section is all different info about project wide things (such as this meeting) so "structure" feels wrong.

I don't think that "hiding" this under "About" is a good idea.

Reply:
If `sunpy.org` is a landing page for the project, I thought having a "SunPy Project" item was a bit redundant. Of the current dropdown options under this heading I'd say:

- https://sunpy.org/project/ definitely belongs in "About" .
- https://sunpy.org/project/roles belongs in "About"
- https://sunpy.org/project/affiliated in a modified version belongs in "Documentation", I think under an "Affiliated packages" page that links to all the package docs (instead of listing all of the packages in the documentaiton drop down)
- https://sunpy.org/project/former belongs at the bottom of the "Roles page"
- https://sunpy.org/project/meetings belongs under "About"

That seems to be a lot of items under "About" now I've listed them... but I'm not sure what the different purposes of "About" and "Project" are or would be?

Reply:
I am not convinced that people would click on "About" to get to "Meetings" (they also probably wouldn't click on "Project" either).

> instead of listing all of the packages in the documentation drop down

I am strongly in favour of keeping all the documentation links 1 click away from any page in the whole sunpy.org tree, so I do not agree that this is a good idea.

So I also don't think that moving the affiliated package page under "documentation" feels right to me either.

Reply:
Maybe lets hack this on Friday with a 🖊️ and 🧻